cmd/appliance/functions/download.go

```diff
@@ -0,0 +1,212 @@
+package functions
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/appgate/sdp-api-client-go/api/v19/openapi"
+	apipkg "github.com/appgate/sdpctl/pkg/api"
+	appliancepkg "github.com/appgate/sdpctl/pkg/appliance"
+	"github.com/appgate/sdpctl/pkg/docs"
+	"github.com/appgate/sdpctl/pkg/factory"
+	"github.com/appgate/sdpctl/pkg/filesystem"
+	"github.com/appgate/sdpctl/pkg/tui"
+	"github.com/appgate/sdpctl/pkg/util"
+	"github.com/hashicorp/go-multierror"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/vbauerster/mpb/v7"
+)
+
+var (
+	logServerImages map[string]string = map[string]string{
+		"cz-opensearch":           "latest",
+		"cz-opensearchdashboards": "latest",
+	}
+	funcImages map[string]map[string]string = map[string]map[string]string{
+		appliancepkg.FunctionLogServer: logServerImages,
+	}
+)
+
+type BundleOptions struct {
```

Comment:
```suggestion
type bundleOptions struct {
```
this struct does not need to be exported